templates/firewall_podnet/configs/nftables.conf.j2

```diff
@@ -50,6 +50,11 @@ table inet firewall_podnet {
     # Traffic TO The PodNet
     chain input {
         type filter hook input priority filter; policy drop;
+
+        # Accept traffic related to established connections
```

Comment:
From my understanding established connections and related connections are two types of connections.

An established connection is where a packet is part of an existing connection. A related connection is a packet that is requesting a new connection but is part of an existing connection.

The comment above reads as if the rule is only for established connections but the rule as accepting both established and related.  

My main concern is the security implications of each especially when updating the firewall ruleset.

Reply:
corrected the comment